### PR TITLE
[RHELC-1102] Enable el9 conversions

### LINKED ISSUE #867: Support for EL9 based clones to convert to RHEL9
Hello, is there also a plan to be able to migrate from EL9 based distributions such as (AlmaLinux, Rocky, Oracle) to RHEL9?

Thank you.

### PR DESCRIPTION
**Pull request by SpyTec**
This is the PR for enabling el9 conversions. This is done through the el9-conversions branch, so if you have anything you want to fix, open a PR to that branch.

Until we get confirmation that everything works fine we will keep this in draft.

Fixes #867, #498.
Follows up on #860.

Jira Issues: [RHELC-1102](https://issues.redhat.com/browse/RHELC-1102)

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [x] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [ ] PR title explains the change from the user's point of view
- [ ] Code and tests are documented properly
- [ ] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -55,6 +57,9 @@ jobs:
         # Even though the distro context is 8.6 we are setting the SYSTEM_RELEASE_ENV to 8.7
         # to tag the updated system correctly
       distros: [centos-8-latest, oraclelinux-8.6]
+    epel-9-x86_64:
+      # TBD: so far it's just guessing of the distro strings
+      distros: [oraclelinux-9-latest, AlmaLinux-OS-9.2, rocky-linux-9.2]
```
Comment (danmyway) [inline .packit.yaml:62]: We will need to get packit/packit-service#2072 resolved somehow, so the check identifiers for EL9 will make sense.
Also, we will call the AMIs from AWS directly in `tf_extra_params`

File: Containerfiles/rpmbuild.oracle9.Containerfile
Comment (pr-watson) [inline Containerfiles/rpmbuild.oracle9.Containerfile:1]: Opting for centos9 container file insteaad of oracle

File: Containerfiles/oracle9.Containerfile
Comment (pr-watson) [inline Containerfiles/oracle9.Containerfile:1]: Opting for centos9 container file instead of oracle

File: requirements/oracle9.requirements.txt
Comment (pr-watson) [inline requirements/oracle9.requirements.txt:1]: Opting for centos9 requirements instead of oracle9

File: .gitignore
Comment (pr-watson) [inline .gitignore:1]: Already merged in main

**Comment by SpyTec** [comment]
We have merged most of these changes to main, hence closing this; el9 conversions are not still in a stable state from our POV and won't be built for releases at the moment. Conversions should work for the ideal setups, contributions welcome!

**Pull request closed without merging**